pages/expense.tsx

```diff
@@ -24,17 +23,26 @@ import MobileCollectiveInfoStickyBar from '../components/expenses/MobileCollecti
 import { Box, Flex } from '../components/Grid';
 import Page from '../components/Page';
 
-export const getVariableFromProps = props => {
+export const getVariablesFromQuery = query => {
```

Comment:
I initially applied your suggestion, but CI reminded me that I kept it this way because another file that doesn't have access to `context` (only to query) depends on this